documentation/buses.yml

```diff
@@ -159,6 +159,54 @@ paths:
                         properties:
                           count:
                             type: number
+  /buses/link-bus-with-route:
+    post:
+      parameters:
+        - name: "Accept-Language"
+          in: "header"
+          default: en
+          required: true
+          description: Preferred language
+      tags:
+        - Buses
+      description: Assign a route to bus
+      security:
+        - bearerAuth: []
+      requestBody:
+        description: Request Body
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/LinkBusToRoute'
+      responses:
+        201:
+          description: Route assigned to bus sucessfully
+  /buses/buses-and-routes-assignments:
```

Comment:
I think `/buses/buses-and-routes-assignments` is a little bit too long. Would you consider maybe renaming it to something like `/buses/routes-assignments`, since we're already on the buses route?